EthOn.rdf

```diff
@@ -1935,9 +1935,10 @@
 
     <owl:Class rdf:about="http://ethon.consensys.net/ExternalAccount">
         <rdfs:subClassOf rdf:resource="http://ethon.consensys.net/Account"/>
+        <simpleDefinition>An Ethereum Account usually controlled by a person. It is identified by an address (0x...) and a pubblic/private key-pair.</simpleDefinition>
         <suggestedStringRepresentation xml:lang="en">ExternalAccount</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">An Account owned by an External Actor.</rdfs:comment>
-        <rdfs:label xml:lang="en">Externally controlled Account</rdfs:label>
+        <rdfs:comment xml:lang="en">One of the two main types of Accounts on Ethereum, an External Account is an Account controlled by an External Actor (as opposed to a Contract Account). The unique identifier of an External Account is its address. It consists of the rightmost 160 bits of the keccak 256 hash of its public key (Example: 0x1a1138875dAB9E0AC8b0fd4a37EEAC5eb26B870B). There is only one way External Accounts can actively change the state of the blockchain. They can be used to send a valid Transaction to the Ethereum blockchain. The transaction can be directed towards other External Accounts or towards Contract Accounts. Since the private key is needed for sending, a valid Transaction can only be created by the person or entity controlling the private key of the External Account. Besides being target of Transactions, External Accounts can also be the target of Contract Messages, Block beneficiary rewards, Uncle beneficiary rewards or payouts of Contract Accounts that have self-destructed.</rdfs:comment>
```

Comment:
- Change capitalization
- Re: `There is only one way External Accounts can actively change the state of the blockchain. They can be used to send a valid Transaction to the Ethereum blockchain.` Is this accurate given the new model Sunny introduced, which groups all Message subclasses (`Transaction`, `CreateMessage`, `ContractMessage` and `CallMessage`) under the Message class? Since an External Actor can create contracts and call contract functions, I'm not sure if this is true.

The new Message scheme is currently only in the Dev branch